libs/ui/elements/src/lib/downloads-list/downloads-list.component.ts

```diff
@@ -75,10 +99,16 @@ export class DownloadsListComponent {
       this.links.some((link) => this.isLinkOfFormat(link, format))
     )
   }
-
+  forceRefreshPagination() {
+    setTimeout(() => {
```

Comment:
this looks like a hack (using timeout), is there another way to trigger changes?